### DISCUSSION TITLE
Dataset for training

### DISCUSSION THREAD

**Discussion by AlbertoPresta** [InterDigitalInc/CompressAI#193] [Q&A]
Hi,

For training your models did you use the entire VImeo90K (The original training + test set [consisting of 91701 sequences, which are not downsampled or downgraded by noise]) [zip (82GB)] or just a portion of it? 

Thanks in advance for the answer, 
Alberto

**Comment by fracape** [comment]
Hi, we randomly pre-selected patches, noting that it did not have a sensible impact compared to the full dataset. Moreover, that was enough to reach the performances from the original papers (for still image coding).
